dotnet-interactive/Lsp/LspSerializer.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+using Newtonsoft.Json.Linq;
+using Newtonsoft.Json.Serialization;
+
+namespace Microsoft.DotNet.Interactive.App.Lsp
+{
+    public static class LspSerializer
+    {
+        public static readonly JsonSerializerSettings JsonSerializerSettings;
+
+        public static JsonSerializer JsonSerializer { get; }
+
+        static LspSerializer()
+        {
+            JsonSerializerSettings = CreateDefaultSettings();
+            JsonSerializer = JsonSerializer.Create(JsonSerializerSettings);
+        }
+
+        private static JsonSerializerSettings CreateDefaultSettings()
+        {
+            var settings = new JsonSerializerSettings
```

Comment:
can we store this value?

Reply:
(See comment below about the `.Error` handler.)